### PR TITLE
Add project button

### PR DESCRIPTION
**Pull request by VolkerJacobsen**

### CODE REVIEW

File: components/ProjectForm/ProjectForm.js [hunk 1]
```diff
@@ -80,68 +67,44 @@ export default function ProjectForm({ onAddProject }) {
...
-    }
   };
-
   return (
     <div>
-      <h1>Publish your project</h1>
       <Form onSubmit={handleSubmit}>
         <Fieldset>
-          <label>
+          <label htmlFor="name">
```
Comment (tejue) [inline components/ProjectForm/ProjectForm.js]: Not sure if I'm mixing up something but htmlFor in label und id in your input should have the same name/word

File: components/ProjectForm/ProjectForm.js [hunk 2]
```diff
@@ -80,68 +67,44 @@ export default function ProjectForm({ onAddProject }) {
...
-          </div>
-          <label>
+          <label htmlFor="imageSource">
             <p>Image: </p>
             <input
               type="file"
               name="imageSource"
               id="imageSource"
               accept="image/*"
               onChange={handleImageChange}
```
Comment (tejue) [inline components/ProjectForm/ProjectForm.js:109]: I learned: if you have an onChange, you always want to have a value as well :)

File: components/ProjectForm/ProjectForm.js [hunk 3]
```diff
@@ -80,68 +67,44 @@ export default function ProjectForm({ onAddProject }) {
...
-              required
             />
           </label>
-          <label>
+          <label htmlFor="shortDescription">
```
Comment (tejue) [inline components/ProjectForm/ProjectForm.js:112]: I think you don't really need an extra p-tag but could instead put this text in the label. The p-Tag you could do just via styling. I think you did this before as well.

When putting text with the label, you can click on it and it leads you directly to the inputfield. this is also good for screenreaders

File: components/ProjectForm/ProjectForm.js [hunk 4]
```diff
@@ -152,7 +115,7 @@ export default function ProjectForm({ onAddProject }) {
               placeholder="Enter a short description. Max. 100 characters."
             />
           </label>
-          <label>
+          <label htmelFor="longDescription">
             <p>Long description: </p>
```
Comment (tejue) [inline components/ProjectForm/ProjectForm.js:124]: same as above

File: components/ProjectForm/ProjectForm.js [hunk 5]
```diff
@@ -165,15 +128,13 @@ export default function ProjectForm({ onAddProject }) {
...
-          <label>
-            Select a category:
-            <Select name="category" id="category" required>
-              <option value="Community">Community</option>
-              <option value="Environment">Environment</option>
-              <option value="Politics">Politics</option>
-            </Select>
-          </label>
-          <label>
+          <label htmlFor="category">Select a category: </label>
```
Comment (tejue) [inline components/ProjectForm/ProjectForm.js]: yes, this is what I meant above :)

File: components/ProjectForm/ProjectForm.js [hunk 6]
```diff
@@ -165,15 +128,13 @@ export default function ProjectForm({ onAddProject }) {
...
+          <Select name="category" id="category" required>
+            <option value="Community">Community</option>
+            <option value="Environment">Environment</option>
+            <option value="Politics">Politics</option>
```
Comment (tejue) [inline components/ProjectForm/ProjectForm.js:142]: I would rather write the value in small letters: community and so on

File: components/ProjectForm/ProjectForm.js [hunk 7]
```diff
@@ -165,15 +128,13 @@ export default function ProjectForm({ onAddProject }) {
...
+          </Select>
+          <label htmlFor="organizer">
             <p>Organizer: </p>
```
Comment (tejue) [inline components/ProjectForm/ProjectForm.js]: see above

File: components/ProjectForm/ProjectForm.js [hunk 8]
```diff
@@ -183,7 +144,7 @@ export default function ProjectForm({ onAddProject }) {
               placeholder="Enter your organization"
             />
           </label>
-          <label>
+          <label htmlFor="contact">
             <p>Contact email: </p>
```
Comment (tejue) [inline components/ProjectForm/ProjectForm.js:155]: see above

File: pages/category/[slug].js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+  const currentProject = projects.find((project) => project.slug === slug);
+
+  if (!currentProject) {
+    return <div>Project not found</div>;
+  }
+  const category = currentProject.category;
+
+  console.log(slug);
+  console.log(category);
+  console.log(currentProject);
```
Comment (tejue) [inline pages/category/[slug].js]: please clean up :)

File: pages/category/[slug].js [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+  return (
+    <div>
```
Comment (tejue) [inline pages/category/[slug].js]: Do you actually really need that many div's? I am not such a div-fan and also it is not very semantic :(

**Review by tejue — COMMENTED** [review]
Hey Volker, nice work.
I really like your app-idea. I found some stuff, mostly concerning semantics. But besides that. Nicely done :rocket:

File: pages/category/[slug].js (hunk 9 above)
Comment (VolkerJacobsen) [reply pages/category/[slug].js]: Thanks! Did so.

File: components/ProjectForm/ProjectForm.js (hunk 1 above)
Comment (VolkerJacobsen) [reply components/ProjectForm/ProjectForm.js]: You are right! Thanks for noticing.

File: components/ProjectForm/ProjectForm.js (hunk 6 above)
Comment (VolkerJacobsen) [reply components/ProjectForm/ProjectForm.js:142]: No, I need the value in Capital letters. ;)

File: components/ProjectForm/ProjectForm.js (hunk 3 above)
Comment (VolkerJacobsen) [reply components/ProjectForm/ProjectForm.js:112]: The click on the label already leads you directly to the input field/ file upload. I will leave it this way.

File: components/ProjectForm/ProjectForm.js (hunk 2 above)
Comment (VolkerJacobsen) [reply components/ProjectForm/ProjectForm.js:109]: I think that is not correct. As far as I know I don´t need to set an initial value. The onChange function will be triggered whenever the user selects a file, and the file will be handled in the handleImageChange function.

File: components/ProjectForm/ProjectForm.js (hunk 4 above)
Comment (VolkerJacobsen) [reply components/ProjectForm/ProjectForm.js:124]: See above

File: components/ProjectForm/ProjectForm.js (hunk 8 above)
Comment (VolkerJacobsen) [reply components/ProjectForm/ProjectForm.js:155]: See above

File: public/placeholder-image.jpg
Comment (matschi3) [inline public/placeholder-image.jpg:1]: keep in mind: sometimes when renaming files, they are not pushed correct to gitHub and vercel throws errors. I think it is primary when u are capitalizing/decapitalizing files. So i think you will not get an error.
yeah just wanted to let you know 😎✌🏼

**Review by matschi3 — APPROVED** [review]
I have nothing to assume here. 🚀

Just left you a little comment on renaming and for some cases you get errors on vercel deployment

**Pull request merged**